client/src/pages/Browse.jsx

```diff
@@ -103,9 +106,9 @@ const Browse = () => {
               </div>
 
               {/* hero text */}
-              {!loading && content && content.length !== 0 && (
+              {!loading && allContent && allContent.length !== 0 && (
```

Comment:
Instead of something like !loading, can we not do {loading && return Loading...} and do the rest next?

Reply:
Done. ThanksQ!